projects/Cryptpad/default.nix

```
@@ -23,7 +23,13 @@
     cryptpad = {
       name = "cryptpad";
       module = "${sources.inputs.nixpkgs}/nixos/modules/services/web-apps/cryptpad.nix";
-      examples.basic = null;
+      examples.basic = {
```

```suggestion
      examples.demo = {
```

I had this idea to reserve the `demo` name for service demos, as they would be rendered differently in the overview than other examples. What do you think?

---

Yup, that's a great idea! I like it.